### PR TITLE
OSDOCS-4632: Observing network traffic

### PR DESCRIPTION
**Pull request by snarayan-redhat**
Version(s): 4.12+

Issue: https://issues.redhat.com/browse/OSDOCS-4632

Link to docs preview: https://54085--docspreview.netlify.app/openshift-enterprise/latest/networking/network_observability/observing-network-traffic.html

QE review:
- [x] QE has approved this change.

QE and SME approval present in #53362 , #53366 , #53405 and #53954

### CODE REVIEW

File: modules/network-observability-topology.adoc [hunk 1]
```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: CONCEPT
+[id="network-observability-topology_{context}"]
+= Observing the network traffic by using Topology
```
Comment (jeana-redhat) [inline modules/network-observability-topology.adoc:6]: Same as the others
```suggestion
= Observing the network traffic from the Topology view
```

File: modules/network-observability-topology.adoc [hunk 2]
```diff
@@ -0,0 +1,7 @@
...
+Topology provides a graphical representation of the network flows and the amount of traffic. As an administrator, you can monitor the traffic data across the application by using the topology view.
```
Comment (jeana-redhat) [inline modules/network-observability-topology.adoc:7]:
```suggestion
The *Topology* view provides a graphical representation of the network flows and the amount of traffic. As an administrator, you can monitor the traffic data across the application by using the *Topology* view.
```

File: modules/network-observability-working-with-topology.adoc [hunk 3]
```diff
@@ -0,0 +1,12 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: PROCEDURE
+[id="network-observability-working-with-topology_{context}"]
+= Working with Topology
```
Comment (jeana-redhat) [inline modules/network-observability-working-with-topology.adoc:6]:
```suggestion
= Working with the Topology view```

File: modules/network-observability-working-with-topology.adoc [hunk 4]
```diff
@@ -0,0 +1,12 @@
...
+As an administrator, you can navigate to topology view by executing the following:
```
Comment (jeana-redhat) [inline modules/network-observability-working-with-topology.adoc:7]:
```suggestion
As an administrator, you can navigate to the *Topology* view to see the details and metrics of the component.
```

File: modules/network-observability-working-with-topology.adoc [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+
+. On the left navigation panel, click *Observe* → *Network Traffic*.
+. In the *Network Traffic* page, click the *Topology* tab.
+
+You can click each component in the topology to view the details and metrics of the component.
```
Comment (jeana-redhat) [inline modules/network-observability-working-with-topology.adoc:12]:
```suggestion
You can click each component in the *Topology* to view the details and metrics of the component.
```

File: modules/network-observability-configuring-options-topology.adoc [hunk 6]
```diff
@@ -0,0 +1,20 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: REFERENCE
+[id="network-observability-configuring-options-topology_{context}"]
+= Configuring the advanced options
+You can customize and export the view by using *Show advanced options*. The advanced options has following features:
```
Comment (jeana-redhat) [inline modules/network-observability-configuring-options-topology.adoc:7]:
```suggestion
You can customize and export the view by using *Show advanced options*. The advanced options view has the following features:
```

File: modules/network-observability-quickfilter.adoc [hunk 7]
```diff
@@ -0,0 +1,29 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: REFERENCE
+[id="network-observability-quickfilter{context}"]
+= Filtering the network traffic
+By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the FlowCollector instance. You can use the filter options to observe the required data by changing the preset filter. Following are the available filter options:
```
Comment (jeana-redhat) [inline modules/network-observability-quickfilter.adoc:7]:
```suggestion
By default, the *Network Traffic* page displays the traffic flow data in the cluster based on the default filters configured in the `FlowCollector` instance. You can use the filter options to observe the required data by changing the preset filter.
```

File: modules/network-observability-quickfilter.adoc [hunk 8]
```diff
@@ -0,0 +1,29 @@
...
+
+
+.Quick filters
+The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 
+
+.Advanced filters
+You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
```
Comment (jeana-redhat) [inline modules/network-observability-quickfilter.adoc:14]: Suggestion: you might like using a [description list](https://docs.asciidoctor.org/asciidoc/latest/lists/description/#basic-description-list) here
```suggestion
Quick filters:: The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console. 

Advanced filters:: You can set the advanced filters by providing the parameter to be filtered and its corresponding text value. The section *Common* in the parameter drop-down list filters the results that match either *Source* or *Destination*. To enable or disable the applied filter, you can click on the applied filter listed below the filter options.
```

File: modules/network-observability-quickfilter.adoc [hunk 9]
```diff
@@ -0,0 +1,29 @@
...
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: REFERENCE
+[id="network-observability-quickfilter{context}"]
+= Filtering the network traffic
+By default, the Network Traffic page displays the traffic flow data in the cluster based on the default filters configured in the FlowCollector instance. You can use the filter options to observe the required data by changing the preset filter. Following are the available filter options:
+
+
+.Quick filters
+The default values in *Quick filters* drop-down are defined in the `FlowCollector` config. You can modify the options from console.
```
Comment (jeana-redhat) [inline modules/network-observability-quickfilter.adoc:11]:
```suggestion
The default values in *Quick filters* drop-down menu are defined in the `FlowCollector` configuration. You can modify the options from console. 
```

File: modules/network-observability-quickfilter.adoc [hunk 10]
```diff
@@ -0,0 +1,29 @@
...
+[NOTE]
+====
+To understand the rules of specifying the text value, click *Learn More*.
+====
+
+
+.Query Options
+You can use *Query Options* to optimize the search results, as listed below:
+
+** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for overview and topology view. The default selected value is *Destination*.
```
Comment (jeana-redhat) [inline modules/network-observability-quickfilter.adoc:25]:
```suggestion
** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for *Overview* and *Topology* views. The default selected value is *Destination*.
```

File: modules/network-observability-quickfilter.adoc [hunk 11]
```diff
@@ -0,0 +1,29 @@
...
+** *Match filters*: You can determine the relation between different filter parameters selected in advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
```
Comment (jeana-redhat) [inline modules/network-observability-quickfilter.adoc:26]:
```suggestion
** *Match filters*: You can determine the relation between different filter parameters selected in the advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
```

File: modules/network-observability-quickfilter.adoc [hunk 12]
```diff
@@ -0,0 +1,29 @@
...
+** *Limit*: The limit for internal backend queries. Depending upon the matching and filter settings, several queries can be performed under the cover, each with this limit set, resulting in more results after aggregation.
+
+You can click *Reset default filter* to remove the existing filters, and apply the filter defined in `FlowCollector` config.
```
Comment (jeana-redhat) [inline modules/network-observability-quickfilter.adoc:29]:
```suggestion
You can click *Reset default filter* to remove the existing filters, and apply the filter defined in `FlowCollector` configuration.
```

File: modules/network-observability-quickfilter.adoc [hunk 13]
```diff
@@ -0,0 +1,29 @@
...
+To understand the rules of specifying the text value, click *Learn More*.
+====
+
+
+.Query Options
+You can use *Query Options* to optimize the search results, as listed below:
+
+** *Reporter Node*: Every flow can be reported from both source and destination nodes. For cluster ingress, the flow is reported from the destination node and for cluster egress, the flow is reported from the source node. You can select either *Source* or *Destination*. The option *Both* is disabled for overview and topology view. The default selected value is *Destination*.
+** *Match filters*: You can determine the relation between different filter parameters selected in advanced filter. The available options are *Match all* and *Match any*. *Match all*  provides results that match all the values, and *Match any* provides results that match any of the values entered. The default value is *Match all*.
+** *Limit*: The limit for internal backend queries. Depending upon the matching and filter settings, several queries can be performed under the cover, each with this limit set, resulting in more results after aggregation.
```
Comment (jeana-redhat) [inline modules/network-observability-quickfilter.adoc:27]: Not sure what "under the cover" means here

File: modules/network-observability-configuring-options-trafficflow.adoc [hunk 14]
```diff
@@ -0,0 +1,18 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: PROCEDURE
+[id="network-observability-configuring-options-trafficflow_{context}"]
+= Configuring advanced options
+You can customize and export the view by using *Show advanced options*.
+You can set the row size by using the *Display options* dropdown. The default value is *Normal*.
```
Comment (jeana-redhat) [inline modules/network-observability-configuring-options-trafficflow.adoc:8]:
```suggestion
You can set the row size by using the *Display options* drop-down menu. The default value is *Normal*.
```

**Review by jeana-redhat — COMMENTED** [review]
Most comments are just small items, like missing metadata, asciidoc formatting stuff, and suggestions to remove self-referential language. The content itself looks really good and is a nice clear rundown of these console views :)

One note on the navigation steps ("On the left navigation panel, ..."): we are instructed to avoid directional language in docs. I talked to Olivia about this and she suggested either saying "Topology view" (apparently that is what this area is called officially) or omitting "left navigation panel" altogether.

/remove-label peer-review-in-progress
/remove-label peer-review-needed
/label peer-review-done

File: modules/network-observability-overview.adoc [hunk 15]
```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: CONCEPT
+[id="network-observability-overview_{context}"]
+= Observing the network traffic by using Overview
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/network-observability-overview.adoc:6]: yes, that's right.

File: modules/network-observability-quickfilter.adoc (hunk 8 above)
Comment (snarayan-redhat) [reply modules/network-observability-quickfilter.adoc:14]: aah yes! That fits in better. Thanks for the suggestion!

File: modules/network-observability-quickfilter.adoc (hunk 13 above)
Comment (snarayan-redhat) [reply modules/network-observability-quickfilter.adoc:27]: It was supposed to mean backend. But I have removed it now as the phrase doesn't make any relevance to the readers.

File: modules/network-observability-trafficflow.adoc [hunk 16]
```diff
@@ -0,0 +1,7 @@
+// Module included in the following assemblies:
+//
+// network_observability/observing-network-traffic.adoc
+:_content-type: CONCEPT
+[id="network-observability-trafficflow_{context}"]
+= Observing the network traffic by using Traffic flows
+Traffic flow provides the data of the network flows and the amount of traffic. As an administrator, you can monitor the amount of traffic across the application by using the traffic flow table.
```
[the remark this answers is not in the document]
Comment (snarayan-redhat) [reply modules/network-observability-trafficflow.adoc:7]: So here, Traffic flows is the tab where the traffic flow data is represented in a tabular format. This tab is one of the views of how we can see the data, tabular format being one of them. I have reworded the sentence accordingly.

**Pull request merged**